README.md

```diff
@@ -145,26 +145,95 @@ If you want to disable the error reporting you can do this by setting the state
+        "es": "Estas notificaciones son recopiladas por el sistema ioBroker y señalan problemas que debe verificar y solucionar.",
+        "pl": "Te powiadomienia są zbierane przez system ioBroker i wskazują problemy, które należy sprawdzić i naprawić.",
+        "zh-cn": "这些通知由ioBroker系统收集，并指出您应检查并修复的问题"
+      },
+      "categories": [
+        {
+          "category": "memIssues",
+          "name": {
+            "en": "Issues with RAM availability",
+            "de": "Probleme mit der Arbeitsspeicher-Verfügbarkeit",
+            "ru": "Проблемы с доступностью оперативной памяти",
+            "pt": "Problemas com disponibilidade de RAM",
+            "nl": "Problemen met de beschikbaarheid van RAM",
+            "fr": "Problèmes de disponibilité de la RAM",
+            "it": "Problemi con la disponibilità della RAM",
+            "es": "Problemas con la disponibilidad de RAM",
+            "pl": "Problemy z dostępnością pamięci RAM",
+            "zh-cn": "RAM可用性问题"
+          },
+          "severity": "alert",
+          "description": {
+            "en": "Your system is running out of memory. Please check the number of running adapters and processes or if single processes need too many memory.",
+            "de": "Es steht nicht genug Arbeitsspeicher zur Verfügung. Die Anzahl der laufenden Adapter und Prozesse sollte geprüft werden, oder ob einzelne Prozesse zuviel Speicher benötigen.",
+            "ru": "В вашей системе не хватает памяти. Пожалуйста, проверьте количество работающих адаптеров и процессов, или если отдельным процессам требуется слишком много памяти.",
+            "pt": "Seu sistema está ficando sem memória. Verifique o número de adaptadores e processos em execução ou se processos únicos precisam de muita memória.",
+            "nl": "Uw systeem heeft onvoldoende geheugen. Controleer het aantal actieve adapters en processen of als afzonderlijke processen te veel geheugen nodig hebben.",
+            "fr": "Votre système manque de mémoire. Veuillez vérifier le nombre d'adaptateurs et de processus en cours d'exécution ou si des processus uniques nécessitent trop de mémoire.",
+            "it": "Il tuo sistema sta esaurendo la memoria. Controllare il numero di adattatori e processi in esecuzione o se i singoli processi richiedono troppa memoria.",
+            "es": "Su sistema se está quedando sin memoria. Verifique el número de adaptadores y procesos en ejecución o si los procesos individuales necesitan demasiada memoria.",
+            "pl": "W Twoim systemie kończy się pamięć. Sprawdź liczbę działających adapterów i procesów lub czy pojedyncze procesy wymagają zbyt dużej ilości pamięci.",
+            "zh-cn": "您的系统内存不足。请检查正在运行的适配器和进程的数量，或者单个进程是否需要太多内存。"
+          },
+          "regex": [
+            "^Exception-Code: ENOMEM",
+            "buffer allocation failed"
+          ],
+          "limit": 10
+        }
+      ]
+    }
+  ]
+}
+```
 
```

Comment:
I propose to add an additional section/sentences like:

Each adapter can define it's own "scopes" for own notifiations with it's own categories which then will be available in the system. Please contact the core development group if you plan to add an own scoe so that scope names can be checked to stay unique.
The same applies if you see the need to enhance the system scope by additional categories. Let's discuss the requirements that they can also be added officially into upcoming js-controller versions.

A scope and category definition needs to contain all relevant UI texts in the JSON.